packages/synapse-react-client/src/components/EntityLink.tsx

```diff
@@ -38,9 +38,13 @@ export const EntityLink = (props: EntityLinkProps) => {
     entityId = entityOrId
   }
 
-  const { data: fetchedEntity, error } = useGetEntity(entityId, versionNumber, {
-    enabled: !!entityId && typeof entityOrId === 'string',
-  })
+  const { data: fetchedEntity, error } = useGetEntityHeader(
+    entityId,
+    versionNumber,
+    {
+      enabled: !!entityId && typeof entityOrId === 'string',
+    },
+  )
```

Comment:
We only need EntityHeader data here.

In a future change, this query's cache data will be pre-loaded by a batch request before SynapseTable renders many instances of this component.